.github/workflows/main.yml

```diff
@@ -111,8 +111,8 @@ jobs:
           cp .buildkite_win/appxmanifest.xml kolibri-windows/
           & "${Env:ProgramFiles(x86)}\Windows Kits\10\App Certification Kit\makeappx.exe" pack /d kolibri-windows /p endlesskeyapp.unsigned.appx
 
-      - name: Sign artifact for release
-        if: startsWith(github.ref, 'refs/tags/')
+      - name: Sign artifact
+        if: ${{ startsWith(github.ref, 'refs/tags/') || github.event_name == 'workflow_dispatch' }}
```

Comment:
We have different signing policies (so they use different certificates) for test builds and release builds. We don't want to sign a test build with our release certificate to avoid having a signed build that could potentially cause problems to users. So if we use the same workflow, we will either need a separate step or to parametrize the signing policy.

Reply:
@starnight Inspired by your work here I opened https://github.com/endlessm/kolibri-app/pull/40, that adds an input parameter to decide whether we should sign with the release or testing policy. Please have a look at that one and let me know what you think.

Reply:
ok